src/bindings/python/tests/test_runtime/test_properties.py

```diff
@@ -247,6 +247,16 @@ def test_properties_ro(ov_property_ro, expected_value):
                 (2.0, 2.0),
             ),
         ),
+        (
+            properties.intel_auto.enable_startup_fallback,
+            "ENABLE_STARTUP_FALLBACK",
+            ((True, True),),
```

Comment:
it is better to add all cases like 
            (
                (True, True),
                (False, False),
                (1, True),
                (0, False),
            ),

Reply:
Okay.